bot/resources/tags/on-message-event.md

```diff
@@ -0,0 +1,16 @@
+---
+embed:
+    title: "The discord.py `on_message` event"
+---
+
+Registering the `on_message` event with [`@bot.event`](https://discordpy.readthedocs.io/en/stable/ext/commands/api.html#discord.ext.commands.Bot.event) will override the default behavior of the event. This may cause prefix commands to stop working, because they rely on the default `on_message` event handler.
+
+Instead, use [`@bot.listen`](https://discordpy.readthedocs.io/en/stable/ext/commands/api.html#discord.ext.commands.Bot.listen) to add a listener. Listeners get added alongside the default `on_message` event, rather than overriding it, so prefix commands can still be invoked as usual:
```

Comment:
I'd also like to include the fact that you can have more than one listener per event, unlike the event decorator
```suggestion
Instead, use [`@bot.listen`](https://discordpy.readthedocs.io/en/stable/ext/commands/api.html#discord.ext.commands.Bot.listen) to add a listener. Listeners get added alongside the default `on_message` event, rather than overriding it, so prefix commands can still be invoked as usual and you can have more than one handler for each event type (unlike events). Here's an example:
```

Reply:
This seems a bit redundant at the end talking about having more than one handler for each event. The beginning of the sentence states listeners are added alongside. Though it isn't a big difference, and the information could be helpful.

Reply:
Maybe we could combine the first and last part?
"Listeners get added alongside the default `on_message` which means you can have multiple handlers for the same event. This means prefix commands can still be invoked as usual. Here's an example:"

Reply:
I think that is a good way of phrasing it.